pip/requirements.txt

```diff
@@ -140,9 +141,9 @@ networkx==2.4 ; python_version>'3.0'
 neurolab==0.3.5
 nose-parameterized==0.6.0
 nose==1.3.7
-notebook==5.7.8 ; python_version<'3.0'
+notebook==5.7.10 ; python_version<'3.0'
 notebook==6.0.2 ; python_version>'3.0'
-numba==0.50.0dev0 ; python_version>'3.0'
+numba==0.50.1 ; python_version>'3.0'
```

Comment:
@davidlange6 , numba 0.50.1 was not picked up due to hard coded commit in the https://github.com/cms-sw/cmsdist/blob/d31f5a3c35fca2d12b62afe01eb6441f6840b126/pip/py3-numba.file .

@mrodozov , please check if py3-numba-llvm33  patch is needed for numba 0.50.1?

Reply:
I see this is done. 
anyway, I'm checking out this whole thing to see how it goes with the additional awkward and uproot :/